app/docs/md/learn/practices/testing.md

```diff
@@ -85,3 +85,40 @@ test('Shut down local server', async t => {
 })
 ```
 
+## Component testing
+
+If you like to run functional component tests, we recommend to use [WebdriverIO](https://webdriver.io/) as browser automation framework. The benefits of using an actual browser compared to JSDOM outweigh the minimal speed detriment that comes with spinning up a browser instance. Furthermore WebdriverIO is based on [WebDriver](https://www.w3.org/TR/webdriver/) as browser automation web standard which guarantees you interact with your component as close as possible as a user would do.
+
+To add a WebdriverIO test harness to your project, run:
+
+```shell
+npm init wdio@latest ./
+```
+
+The command initializes a configuration wizard that helps you set-up the test harness. Make sure to select on the first question: "_Where should your tests be launched?_" the answer: "_browser - for unit and component testing in the browser_".
+
+An example test file should be created, replace the content with e.g.:
+
+```javascript
+import { expect, browser, $$ } from '@wdio/globals'
+import enhance from '@enhance/ssr'
+
+// see actual component here: https://github.com/webdriverio/component-testing-examples/blob/main/enhance/app/elements/my-header.mjs
+import MyHeader from '../../app/elements/my-header.mjs'
+
+describe('Enhance Framework', () => {
+  it('should render MyHeader element correctly', async () => {
+    const html = enhance({
+      elements: {
+        'my-header': MyHeader
+      }
+    })
+    const actual = document.createElement('div')
+    actual.innerHTML = (html`<my-header></my-header>`).replace(/<(\/*)(html|head|body)>/g, '')
+    document.body.appendChild(actual)
+    expect(await $$('img').length).toBe(2)
+  })
+})
+```
+
+You can find the WebdriverIO API docs at [webdriver.io/docs/api](https://webdriver.io/docs/api) and the full example for Enhance component testing [on GitHub](https://github.com/webdriverio/component-testing-examples/tree/main/enhance). If you have any questions, join the projects [Gitter Channel](https://gitter.im/webdriverio/webdriverio).
```

Comment:
```suggestion
You can find the WebdriverIO API docs at [webdriver.io/docs/api](https://webdriver.io/docs/api) and the full example for Enhance component testing [on GitHub](https://github.com/webdriverio/component-testing-examples/tree/main/enhance). If you have any questions, join the project's [Gitter channel](https://gitter.im/webdriverio/webdriverio).
```

(By the way, Given [Gitter's impending migration to Matrix](https://blog.gitter.im/2023/01/16/gitter-is-going-fully-native-matrix-in-feb-2023/), this link should probably be changed to the new URL scheme once that's available, which should happen next Monday.)